ambry-api/src/main/java/com/github/ambry/account/Container.java

```diff
@@ -487,6 +508,7 @@ JSONObject toJson() throws JSONException {
         metadata.put(Container.JSON_VERSION_KEY, JSON_VERSION_2);
         metadata.put(CONTAINER_ID_KEY, id);
         metadata.put(CONTAINER_NAME_KEY, name);
+        metadata.put(CONTAINER_LAST_UPDATE_TIME_KEY, lastUpdateTime);
```

Comment:
Please don't change the current version. Please add JSON_VERSON_3 and only in version_3 can we put container_last_update_time_key.

Reply:
I think we should not create a version 3. version 2 was only created since we removed a field between v1 and v2.

However, you are right that adding a new required field to an existing version will cause issues when deserializing existing records. Generally the best way to make the migration seamless for json documents is to make any new field optional in version 2.

Reply:
Make the new field optional in version2.